tests/test_modeling_tf_common.py

```diff
@@ -1068,10 +1141,13 @@ def _check_match_tokens(self, generated_ids, bad_words_ids):
         return False
 
 
+global_rng = random.Random()
```

Comment:
not sure if we want to have a global random seed, was this an intended change or just used for debugging?

Reply:
Here, I followed the convention from the corresponding PyTorch testing file `test_modeling_common.py`, where `global_rng` is also defined for a similar purpose. However, I think it's okay to omit this change and leave it as it was :)

Reply:
I would be more in favor to keep it as it was.

Reply:
@jplu I removed `global_rng` and leave it as it was before changes. Hopefully, now this PR is ready for a final review